### ISSUE TITLE
Takeoff ✈️

### ISSUE THREAD

**Issue by cha0sg0d** [personaelabs/nym#2]
I'm just starting to go through the repo and wrap my head around how things work.

I got the following issue:

```js
wait  - compiling / (client and server)...
error - ../../node_modules/@personaelabs/spartan-ecdsa/build/helpers/utils.js:39:0
Module not found: Can't resolve 'fs'
```

Steps to reproduce my bug:
1. clone repo
2. Run `yarn install`
3. `cd packages/test_client && npm run dev`

**Comment by lsankar4033** [comment]
I just cloned fresh and ran `yarn && cd packages/test_client && yarn dev` and things worked for me

Which version of node? I'm on 16.14.0 so might need to update

**Comment by cha0sg0d** [comment]
> cd packages/test_client
![grab 2023-03-29 at 16 30 47 jpeg](https://user-images.githubusercontent.com/42984734/228690456-23d1aecb-8cb0-4f9b-8597-c5623372d8e4.jpg)

Super weird, I still get that error on your exact Node version and fresh clone. To humor me, could you try cloning from a fresh repo and see what happens? 

Also re: the specific error, looks like it's coming from [here](https://github.com/personaelabs/spartan-ecdsa/blob/b7e9c32e053a46786b39d3c320a0c7067bfe04cc/packages/lib/src/helpers/utils.ts#L4). I'm wondering how this worked in the past for y'all. I see that the code that uses the `fs` module is explicitly [not used](https://github.com/personaelabs/spartan-ecdsa/blob/b7e9c32e053a46786b39d3c320a0c7067bfe04cc/packages/lib/src/helpers/utils.ts#L22) on web. But perhaps it's still giving an import error... weird that you don't have that problem though.

**Comment by DanTehrani** [comment]
@cha0sg0d 

Can you change `next.config.js` to this and try again?
```jsx
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  webpack: config => {
    config.resolve.fallback = { fs: false };
    config.experiments = { asyncWebAssembly: true };

    return config;
  },
}

module.exports = nextConfig
```

**Comment by cha0sg0d** [comment]
This got it working for me in the `next.js.config`
```typescript
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  webpack: (config) => {
    config.resolve.fallback = { fs: false, readline: false };
    config.experiments = { asyncWebAssembly: true };

    return config;
  },
};

module.exports = nextConfig;

```

**Comment by cha0sg0d** [comment]
Branch with the changes are here: https://github.com/emergenceland/nym/tree/cha0s/svelte

**Comment by cha0sg0d** [comment]
Continuing to document some basic behavior.

I successfully connected a wallet, made a nym, and entered text to post. Then, when I confirmed the signing of the post, I got the following error:

![grab 2023-03-29 at 22 51 39 jpeg](https://user-images.githubusercontent.com/42984734/228741913-175e4d04-e85b-43ba-a459-57e64ffc9ef1.jpg)

When I address the bug, which is coming from `@ethereumjs ... fromRpcSig()`, I get a new error in the same function:

![grab 2023-03-29 at 23 04 52 jpeg](https://user-images.githubusercontent.com/42984734/228744325-3ef031a5-d689-4ca3-9899-71405524c277.jpg)

**Comment by cha0sg0d** [comment]
Closed by #22
